src/circle.vertex.glsl

```diff
@@ -6,59 +6,39 @@ precision highp float;
 #define highp
 #endif
 
-#ifndef MAPBOX_GL_JS
-// set by gl_util
-uniform float u_size;
-#else
 uniform mat4 u_matrix;
 uniform vec2 u_extrude_scale;
 uniform float u_devicepixelratio;
-#endif
 
 attribute vec2 a_pos;
 
 #ifndef MAPBOX_GL_JS
-uniform mat4 u_matrix;
-uniform mat4 u_exmatrix;
+uniform float u_radius;
 #else
-#pragma mapbox: define color lowp
 #pragma mapbox: define radius mediump
 #endif
 
 varying vec2 v_extrude;
-#ifdef MAPBOX_GL_JS
 varying lowp float v_antialiasblur;
-#endif
 
 void main(void) {
-#ifdef MAPBOX_GL_JS
-
-    #pragma mapbox: initialize color lowp
```

Comment:
We need to retain this `#pragma` statement for data-driven styling functions to work in GL JS. If `color` needs to be an attribute, some initialization is required in the vertex shader to make the value accessible by the fragment shader.